modules/nw-cudn-cr.adoc

```diff
@@ -28,7 +28,7 @@ $ cat << EOF | oc apply -f -
 apiVersion: v1
 kind: Namespace
 metadata:
-  name: <cudn_namespace_name>      
+  name: <cudn_namespace_name>
```

Comment:
@JoeAldinger As we are adding support for Localnet, I think it would make sense adding it to CUDN CR section.
In the note it says {VirtProductName} only supports the "Layer2" topology but now "Localnet" is also supported.

Reply:
@JoeAldinger It is not in this file itself.
modules/nw-cudn-cr.adoc

Reply:
@JoeAldinger Could you explain why the change is out of scope of this PR and how do you intend to address it in follow up?